### DISCUSSION TITLE
What plugin adds the LSP progress indicators?

### DISCUSSION THREAD

**Discussion by ConnorWay32** [LazyVim/LazyVim#307] [Q&A] [answered]
I wanted to swap out the default LSP progress popups (for loading workspaces, formatting, etc) with fidget.nvim, but I couldn't work out which plugin to disable to get rid of the old ones. I don't know if the plugins on the LazyVim docs are a complete list, but I looked through the full specs and wasn't able to find the one responsible. Is this built into LazyVim, and if so, is there a way to remove it?

**Comment by folke** [comment, marked as answer]
that's part of noice.nvim. Check it's readme and config on how to disable lsp progress.

**Reply by ConnorWay32** [reply]
Thanks, was able to disable it with the following (if anyone has the same question).
```lua
{
  "folke/noice.nvim",
  opts = {
      lsp = {
        progress = {
          enabled = false,
        },
      },
    },
},
```

**Reply by fdelacruz** [reply]
Hi @ConnorWay32, is this what you mean?
<img width="682" alt="Screen Shot 2023-12-15 at 10 11 48 AM" src="https://github.com/LazyVim/LazyVim/assets/148098/50e1d87d-9ea5-4727-bb1d-90eaab15c070">

**Reply by dpetka2001** [reply]
@fdelacruz No, this won't work. You can either create another spec fragment for `noice.nvim` where you do something like 
```lua
{
  "folke/noice.nvim",
  opts = {
    lsp = {
      progress = {
        enabled = false,
      },
    },
  },
},
```
or if you want to reuse the same piece of spec you have in your code already you should put after the `table.insert` function and before the `end` (in line 18 you have `end,` , so create a new empty line just above line 18 and remove lines 10-15) the following
```lua
return vim.tbl_deep_extend("force", opts, {
  lsp = {
    progress = {
      enabled = false,
    },
  },
})
```
I understand this all might seem a bit complicated, but you should learn some basic Lua to understand how to do things yourself. You will need stuff like that if you want to configure, not just LazyVim, but Neovim in general (even if at some point you decide to write your own custom config that is not a distro).

**Reply by rfletchr** [reply]
thanks for posting this!! they're so damn distracting.
